diff_drive_controller/include/diff_drive_controller/diff_drive_controller.h

```diff
@@ -131,6 +143,9 @@ namespace diff_drive_controller{
     boost::shared_ptr<realtime_tools::RealtimePublisher<tf::tfMessage> > tf_odom_pub_;
     Odometry odometry_;
 
+    /// Joint Trajectory Controller State
```

Comment:
I think it'd be neater to not refer to this publisher as `joint_trajectory_controller_state_pub_` as that only refers to the type. This is in fact just controller state in my opinion. Would it be better to refactor it to reflect that?